modules/tinymce/src/plugins/tabfocus/test/ts/browser/TabfocusSanityTest.ts

```diff
@@ -1,54 +1,40 @@
-import { Keys, Log, Logger, Pipeline, Step, Waiter } from '@ephox/agar';
-import { Assert, UnitTest } from '@ephox/bedrock-client';
-import { TinyActions, TinyApis, TinyLoader } from '@ephox/mcagar';
+import { Keyboard, Keys, Waiter } from '@ephox/agar';
+import { describe, it } from '@ephox/bedrock-client';
+import { TinyDom, TinyHooks } from '@ephox/mcagar';
+import { assert } from 'chai';
 
-import TabfocusPlugin from 'tinymce/plugins/tabfocus/Plugin';
+import Editor from 'tinymce/core/api/Editor';
+import Plugin from 'tinymce/plugins/tabfocus/Plugin';
 import Theme from 'tinymce/themes/silver/Theme';
 
-UnitTest.asynctest('browser.tinymce.plugins.tabfocus.TabfocusSanityTest', (success, failure) => {
-
-  Theme();
-  TabfocusPlugin();
-
-  const sAddInputs = (editor) => {
-    return Logger.t('Add inputs', Step.sync(() => {
-      const container = editor.getContainer();
-      const input1 = document.createElement('input');
-      input1.id = 'tempinput1';
+describe('browser.tinymce.plugins.tabfocus.TabfocusSanityTest', () => {
+  const hook = TinyHooks.bddSetupLight<Editor>({
+    plugins: 'tabfocus',
+    tabfocus_elements: 'tempinput1',
+    base_url: '/project/tinymce/js/tinymce'
+  }, [ Theme, Plugin ], true);
 
-      container.parentNode.insertBefore(input1, container);
-    }));
+  const addInputs = (editor: Editor) => {
```

Comment:
Can this please be converted to before/after hooks instead?

- [ ] Convert to before/after hooks instead of manually calling this